src/themes/uclouvain/app/item-page/full/field-components/file-section/upload-file-description/upload-file-description.component.ts

```diff
@@ -51,6 +53,11 @@ export class UploadFileDescriptionComponent implements OnInit, OnDestroy {
             payload.masterPolicy = this.defaultPolicy;
           }
           this.accessConditions = payload;
+        }),
+      this.bitstream.download_url
+        .pipe(getFirstSucceededRemoteDataWithNotEmptyPayload())
+        .subscribe((payload: BitstreamDirectDownloadURL) => {
+          this.downloadUrl = payload.url;
         })
```

Comment:
oneLineCommand --> `.subscribe((payload BitstreamDirectDownloadURL) => this.downloadUrl = payload.url)`